wremnants/combine_theory_helper.py

```diff
@@ -119,60 +122,25 @@ def add_minnlo_scale_uncertainty(self, sample_group, use_hel_hist=False, rebin_p
         # assuming the name is unchanged
         obs = self.card_tool.fit_axes
         pt_ax = "ptVgen" if "ptVgen" not in obs else "ptVgenAlt"
-        charge_ax = "chargeVgen" if "chargeVgen" not in obs else "chargeVgenAlt"
-        sum_axes = [pt_ax, charge_ax]
 
-        syst_axes = sum_axes + ["muRfact", "muFfact"]
-        syst_ax_labels = ["PtV", "genQ", "muR", "muF"]
-        format_with_values = ["low", "center", "center", "center"]
-        if use_hel_hist:
-            syst_axes.insert(2, "helicity")
-            syst_ax_labels.insert(2, "AngCoeff")
-            format_with_values.insert(2, "low")
+        syst_axes = [pt_ax, "vars"]
+        syst_ax_labels = ["PtV", "var"]
+        format_with_values = ["edges", "center"]
 
         group_name = f"QCDscale{self.sample_label(sample_group)}"
-        # Exclude all combinations where muR = muF = 1 (nominal) or where
-        # they are extreme values (ratio = 4 or 1/4)
-        skip_entries = [{"muRfact" : 1.j, "muFfact" :  1.j}, {"muRfact" : 0.5j, "muFfact" : 2.j}, 
-                        {"muRfact" : 2.j, "muFfact" : 0.5j}]
-        # In order to make prettier names than the automated ones.
-        # No harm in leaving extra replaces that won't be triggered
-        name_replace = [("muR2muF2", "muRmuFUp"), ("muR0p5muF0p5", "muRmuFDown"), ("muR2muF1", "muRUp"), 
```

Comment:
Are we completely removing these customisation of output histogram names or is it being moved elsewhere (it didn't seem so).